modules/game_bot/default_configs/vBot_4.7/vBot/new_healer.otui

```diff
@@ -0,0 +1,413 @@
+CategoryCheckBox < CheckBox
+  font: verdana-11px-rounded 
+  margin-top: 3
+
+  $checked:
+    color: #98BF64
+
+HealScroll < Panel
+
+  ToolTipLabel
+    id: text
+    anchors.left: parent.left
+    anchors.right: parent.right
+    anchors.top: parent.top
+    text-align: center
+    text: test
+    
+  HorizontalScrollBar
+    id: scroll
+    anchors.left: parent.left
+    anchors.right: parent.right
+    anchors.top: prev.bottom
+    margin-top: 3
+    minimum: 0
+    maximum: 100
+    step: 1
+
+HealItem < Panel
+
+  BotItem
+    id: item
+    anchors.left: parent.left
+    anchors.verticalCenter: parent.verticalCenter
+    size: 34 34
+
+  ToolTipLabel
+    id: text
+    anchors.fill: parent
+    anchors.left: prev.right
+    margin-left: 8
+    text-wrap: true
+    text-align: left
+
+ToolTipLabel < UIWidget
+  font: verdana-11px-rounded
+  color: #dfdfdf
+  height: 14
+  text-align: center 
+
+HealerPlayerEntry < Label
+  background-color: alpha
+  text-offset: 5 1
+  focusable: true
+  height: 16
+  font: verdana-11px-rounded
+  text-align: left
+
+  $focus:
+    background-color: #00000055
+  
+  Button
+    id: remove
+    anchors.right: parent.right
+    margin-right: 2
+    anchors.verticalCenter: parent.verticalCenter
+    size: 15 15
+    margin-right: 15
+    text: X
+    tooltip: Remove player from the list
+
+PriorityEntry < ToolTipLabel
+  background-color: alpha
+  text-offset: 18 1
+  focusable: true
+  height: 16
+  font: verdana-11px-rounded
+  text-align: left
+
+  $focus:
+    background-color: #00000055
+
+  CheckBox
+    id: enabled
+    anchors.left: parent.left
+    anchors.verticalCenter: parent.verticalCenter
+    size: 15 15
+    margin-top: 2
+    margin-left: 3 
+  
+  Button
+    id: increment
+    anchors.right: parent.right
+    margin-right: 2
+    anchors.verticalCenter: parent.verticalCenter
+    size: 14 14
+    text: +
+    tooltip: Increase Priority
+
+  Button
+    id: decrement
+    anchors.right: prev.left
+    margin-right: 2
+    anchors.verticalCenter: parent.verticalCenter
+    size: 14 14
+    text: -
+    tooltip: Decrease Priority
+
+TargetSettings < Panel
+  size: 280 125
+  padding: 3
+  image-source: /images/ui/window
+  image-border: 6
+
+  Label
+    anchors.horizontalCenter: parent.horizontalCenter
+    anchors.top: parent.top
+    font: verdana-11px-rounded 
+    text: Heal Target Settings
+
+  Groups
+    id: groups
+    anchors.top: prev.bottom
+    margin-top: 8
+    anchors.left: parent.left
+    margin-left: 9
+
+  Vocations
+    id: vocations
+    anchors.left: prev.right
+    margin-left: 5
+    anchors.verticalCenter: prev.verticalCenter
+
+Groups < FlatPanel
+  size: 150 90
```

Comment:
```suggestion
  size: 150 100
```

Reply:
![image](https://github.com/mehah/otclient/assets/114332266/27cfbbed-f1f9-4e30-8115-3ee62229d3b8)


with 90 it is not possible to click on the checkbox